File: Java/프로그래머스/0/181837. 커피 심부름/커피 심부름.java
class Solution {
    public int solution(String[] order) {
        int answer = 0;
        for(int i=0; i<order.length ; i++){
            if(order[i].equals("americanoice") || order[i].equals("iceamericano")
               || order[i].equals("americanohot") || order[i].equals("hotamericano")
               || order[i].equals("americano") || order[i].equals("anything"))
                answer+=4500;
            else if(order[i].equals("cafelattehot") || order[i].equals("hotcafelatte")
               || order[i].equals("cafelatteice") || order[i].equals("icecafelatte")
               || order[i].equals("cafelatte"))
                answer+=5000;
        }
        return answer;
    }
}

File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.js
function solution(order) {
    var answer = 0;
    for(let i=0; i<order.length ; i++){
        if(order[i] === ("americanoice") || order[i] === ("iceamericano")
              || order[i] === ("americanohot") || order[i] === ("hotamericano")
              || order[i] === ("americano") || order[i] === ("anything"))
            answer+=4500;
        else if(order[i] === ("cafelattehot") || order[i] === ("hotcafelatte")
               || order[i] === ("cafelatteice") || order[i] === ("icecafelatte")
               || order[i] === ("cafelatte"))
            answer+=5000;
        }
    return answer;
}